<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentexam</class>
 <widget class="QWidget" name="studentexam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2560</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color:white;
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1850</x>
     <y>700</y>
     <width>511</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QRadioButton" name="radioButton_17">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QRadioButton" name="radioButton_18">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QRadioButton" name="radioButton_19">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="radioButton_12">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QRadioButton" name="radioButton_15">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="radioButton_16">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QRadioButton" name="radioButton_20">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="radioButton_7">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QRadioButton" name="radioButton_8">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QRadioButton" name="radioButton_9">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="radioButton_11">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton_1">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QRadioButton" name="radioButton_13">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QRadioButton" name="radioButton_14">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="radioButton_10">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QRadioButton" name="radioButton_21">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QRadioButton" name="radioButton_22">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QRadioButton" name="radioButton_23">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QRadioButton" name="radioButton_24">
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QRadioButton" name="radioButton_25">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>420</y>
     <width>571</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;学号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="indent">
    <number>10</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1790</x>
     <y>1280</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:none;
border-radius:10px;
background-color:rgb(51, 102, 153);
color:white;
}
QPushButton:hover
{
background-color:rgb(177, 204, 228);
color:black;
}</string>
   </property>
   <property name="text">
    <string>上一题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>2140</x>
     <y>1280</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:none;
border-radius:10px;
background-color:rgb(51, 102, 153);
color:white;
}
QPushButton:hover
{
background-color:rgb(177, 204, 228);
color:black;
}</string>
   </property>
   <property name="text">
    <string>下一题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1790</x>
     <y>1410</y>
     <width>651</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border:none;
border-radius:10px;
background-color:rgb(51, 102, 153);
color:white;
}
QPushButton:hover
{
background-color:rgb(177, 204, 228);
color:black;
}</string>
   </property>
   <property name="text">
    <string>交卷</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>1641</width>
     <height>1481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QTextBrowser" name="selectquestion_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-40</y>
       <width>1451</width>
       <height>931</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'宋体'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Microsoft YaHei UI';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>890</y>
       <width>1451</width>
       <height>511</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="chooseA">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chooseB">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>B</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chooseC">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chooseD">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>D</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QTextBrowser" name="textquestion_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>371</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI'; font-size:14pt;&quot;&gt;题目&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="answer">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>370</y>
       <width>521</width>
       <height>221</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Microsoft YaHei UI'; font-size:14pt;&quot;&gt;请作答：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="countdownLabel">
   <property name="geometry">
    <rect>
     <x>1840</x>
     <y>560</y>
     <width>341</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>350</y>
     <width>561</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;姓名：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="indent">
    <number>10</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>2020</x>
     <y>120</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/image/photo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1810</x>
     <y>280</y>
     <width>561</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;班级：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="indent">
    <number>10</number>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>gridLayoutWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>countdownLabel</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
  <zorder>label</zorder>
  <zorder>label_7</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
